### DISCUSSION TITLE
Popup issue with UBO on Brave / Edge not firefox

### DISCUSSION THREAD

**Discussion by share3141** [uBlockOrigin/uBlock-issues#1974] [General] [answered]
I probably screwed up the github site by posting this isue there.  I didn't know about this site. Sorry.
In Brave and Edge, not Firefox, when I click on the UBO icon in the browser and the popup shows up the icon that "Click to Disable Javascript" is not shown. I am using UBO 1.41.2 on Windows 10 21H1 64 bit. I am using Edge Version 98.0.1108.50 (Official build) (64-bit) and Brave [Version 1.36.90 Chromium: 98.0.4758.87 (Official Build) beta (64-bit)](https://brave.com/latest/) and Firefox 98.0b3. 
![Edge](https://user-images.githubusercontent.com/20307002/153646679-e30d2374-8447-43a0-a2db-261d78ff8cc8.png)
![Firefox](https://user-images.githubusercontent.com/20307002/153646681-6e53c5c9-69f7-4e68-8a76-60123c83074e.png)

**Comment by ghost** [comment, marked as answer]
https://github.com/uBlockOrigin/uBlock-issues/issues/1972

**Reply by share3141** [reply]
The issue turned out to be that I had my zoom settings at 150%. I also have my DPI settings at 125%. So I lowered my zoom settings to 125% at the problem disappeared. I'm half blind so I have things cranked up so I can see.  Thanks.
